.circleci/images/primary/Dockerfile-2.2.10

```diff
@@ -50,27 +127,14 @@ RUN set -ex \
   && (docker version || true)
 
 # Install Docker Compose
-RUN COMPOSE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/docker-compose-latest" \
-  && curl --silent --show-error --location --fail --retry 3 --output /usr/bin/docker-compose $COMPOSE_URL \
-  && chmod +x /usr/bin/docker-compose \
-  && docker-compose version
+RUN echo 'deb http://ftp.debian.org/debian stretch-backports main\ndeb-src http://ftp.debian.org/debian stretch-backports main' > /etc/apt/sources.list.d/stretch-backports.list \
+  && apt-get update \
+  && apt-get install -y docker-compose \
+  && docker-compose version \
+  && rm -rf /var/lib/apt/lists/*
 
 # Install Dockerize
-RUN DOCKERIZE_URL="https://circle-downloads.s3.amazonaws.com/circleci-images/cache/linux-amd64/dockerize-latest.tar.gz" \
-  && curl --silent --show-error --location --fail --retry 3 --output /tmp/dockerize-linux-amd64.tar.gz $DOCKERIZE_URL \
-  && tar -C /usr/local/bin -xzvf /tmp/dockerize-linux-amd64.tar.gz \
-  && rm -rf /tmp/dockerize-linux-amd64.tar.gz \
-  && dockerize --version
-
-# Install Protobuf compiler
-RUN export PROTOC_URL="https://github.com/protocolbuffers/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip" \
-  && echo Protoc URL: $PROTOC_URL \
-  && curl --silent --show-error --location --fail --retry 3 --output /tmp/protoc.zip "${PROTOC_URL}" \
-  && ls -lha /tmp/protoc.zip \
-  && unzip /tmp/protoc.zip -d /tmp/protoc \
-  && mv /tmp/protoc/bin/* /usr/bin \
-  && rm -rf /tmp/protoc /tmp/protoc.zip \
-  && protoc --version
+RUN curl -sfL $(curl -s https://api.github.com/repos/powerman/dockerize/releases/latest | grep -i /dockerize-$(uname -s)-$(dpkg --print-architecture)\" | cut -d\" -f4) | install /dev/stdin /usr/local/bin/dockerize
 
 # Install RubyGems
 RUN gem update --system 2.7.9
```

Comment:
Similar to my comment for Ruby 2.1, I've been using rubygems `2.7.11` on 2.2 on my mac without issues for a long time, so I suggest adopting that one here :)

Reply:
I propose we evaluate which version of bundler and rubygems should be used on each ruby version in another PR. Ideally it should be the unconstrained latest of both, but that may not be possible for all versions as we found out with @TonyCTHsu.

Reply:
Done in https://github.com/DataDog/dd-trace-rb/pull/2405